.github/workflows/dapr-pubsub.yaml

```diff
@@ -0,0 +1,65 @@
+name: dapr-pubsub
+on:
+  push:
+    paths:
+      - "pkg/pubsub/dapr"
+
+jobs:
+  dapr_test:
+    name: "Dapr pubsub test"
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    permissions:
+      contents: read
+    strategy:
+      matrix:
+        HELM_VERSION: ["3.7.2"]
+        GATEKEEPER_NAMESPACE: ["gatekeeper-system"]
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
+    - name: Set up Go 1.19
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+
+    - name: Bootstrap e2e
+      run: |
+        mkdir -p $GITHUB_WORKSPACE/bin
+        echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
+        make e2e-bootstrap
+        make e2e-helm-install
+        helm repo add dapr https://dapr.github.io/helm-charts/
+        helm repo update
+        helm upgrade --install dapr dapr/dapr --version=1.10 --namespace dapr-system --create-namespace --wait        
+        helm repo add bitnami https://charts.bitnami.com/bitnami
+        helm install redis bitnami/redis --namespace default
+        make e2e-subscriber-deploy
+
+    - name: Run e2e
+      run: |
+        make docker-buildx IMG=gatekeeper-e2e:latest CRD_IMG=gatekeeper-crds:latest
+        kind load docker-image --name kind gatekeeper-e2e:latest gatekeeper-crds:latest
+        kubectl create ns gatekeeper-system
```

Comment:
As far as I know, pubsub component needs to exist before dapr injects the sidecar into the audit pod. There could be a way to create components later on and still have dapr sidecar use it as intended. But until I figure it out, we can create components beforehand to make sure we have e2e running without error

Reply:
To load the components on daprd sidecar after the pod is up, it needs a restart. To avoid that restart, we need to have component created before pod is created